prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-emails-container/notification-emails-container.component.ts

```diff
@@ -30,10 +32,23 @@ export class NotificationEmailsContainerComponent implements OnInit {
 
   public getTemplateProperties(template: EmailTemplate) {
     return [
+      {
+        key: 'Subject',
+        value: template.subject
+      },
+      {
+        key: 'Description',
+        value: template.description
+      },
+      {
+        key: 'Recipient',
+        value: template.recipient
+      },
       {
         key: 'Last Modified',
-        value: this.formatDatePipe.transform(template.modifiedDate)
-      }
+        value: this.formatDatePipe.transform(template.modifiedDate),
+        danger: DateUtils.withinDaysAfterDate(template.modifiedDate, 182)
```

Comment:
What's the reason for `182` rather than `180`?

Reply:
not sure, I believe I got the number from Claire.

Reply:
it is changed to 180.